app/lib/system/error_reporting.rb

```diff
@@ -12,6 +12,17 @@ def report_error(exception, logger: Rails.logger, **parameters)
       end
     end
 
+    def report_deprecation_warning(payload)
+      message = payload[:message]
+      deprecation_horizon = payload[:deprecation_horizon]
+
+      ::Bugsnag.notify(message) do |report|
+        report.severity = 'warning'
+        report.grouping_hash = message
+        report.add_tab 'deprecation_horizon', { description: "Rails version that won't have this feature available", value: deprecation_horizon }
```

Comment:
mm, I kind of don't like to add this description... 
I mean, it's useful to understand what "deprecation horizon" is, but passing it as a value each time to BugSnag... to me doesn't seem right.
I would maybe do just `{ version: payload[:deprecation_horizon] }` (also, probably not necessary to extract it to a variable if we're just using it one time)

Reply:
Done